Decred Wallet/Features/Settings/SettingsController.swift

```diff
@@ -62,12 +82,34 @@ class SettingsController: UITableViewController  {
         }
     }
     
+    func setMenuTitle(){
+        self.changeSpendingPinPassDesc.text = LocalizedStrings.changeSpendingPinPassDesc
```

Comment:
Can't these be set in IB using `xibLocKey`?

Reply:
it causes runtime error